build.gradle

```diff
@@ -417,7 +418,7 @@ subprojects {
 }
 
 spotless {
-  ratchetFrom 'origin/master'
+  ratchetFrom "${git.getUpstreamRemote()}/main"
```

Comment:
It should be `origin/main` since it runs on every commit, so no need to diff against upstream.

Reply:
Should I use `"origin/${git.getBranch()}"` instead?

Reply:
Discussed offline. We agreed to keep using `"${git.getUpstreamRemote()}/main"`

I tried changing it to: `"${git.upstreamRemote}/main"` but it didn't work